style.css

```diff
@@ -1,30 +1,87 @@
 html * {
   margin: 0;
   padding: 0;
+  font-family: monospace;
+}
+
+header {
+  margin: 20px;
 }
 
 h1 {
   margin: 10px 0;
+  font-size: 30px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
- [ ]  Let's keep our code DRY. align-items: center and flex-direction: column has been repeated more than once. To solve this you can group all classes with the same attribute together and give it one attribute eg
```
.books-container .book-list,
form {
    align-items:center
}
```